packages/backend/src/datasource/datasource.service.spec.ts

```
@@ -138,6 +142,108 @@ describe('DataSourceService', () => {
+
+    expect(createClient).toHaveBeenCalledWith(
+      expect.objectContaining({ plaintextPassword: 'plain-secret' }),
+    );
+  });
+
+  it('tests unsaved settings without touching persistence', async () => {
+    const client = createMockPgClient();
+    client.query.mockResolvedValue({ rows: [], fields: [] });
+
+    jest
+      .spyOn(service as unknown as { createClient: () => unknown }, 'createClient')
```

ESLint

[error] Replace `service·as·unknown·as·{·createClient:·()·=>·unknown·},·'createClient'` with `⏎········service·as·unknown·as·{·createClient:·()·=>·unknown·},⏎········'createClient',⏎······`

(prettier/prettier)